docker-compose.yaml

```diff
@@ -51,4 +62,5 @@ services:
       - ./:/home/node
     ports:
       - "6006:6006"
+    working_dir: /home/node
```

Comment:
nit: This is not required anymore. The Dockerfile has `WORKDIR` set to `/home/node`

Reply:
Aaaah, I see. I probably should have rebuilt the image locally, didn't come to my mind.